### PR TITLE
Welsh Translation changes

### PR DESCRIPTION
**Pull request by DavidJayakumar**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-5497

### Change description ###

Welsh release changes only 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
approved

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/dialogs/document-dialog/document-dialog.component.html [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
   <div class="dialog-header">
     <h2 (click)="cancel()" class="heading-h2 x">X</h2>
   </div>
   <div>
-    <h2 class="heading-h2 dialog-title">Are you sure you want to replace the existing file?</h2>
+    <h2 class="heading-h2 dialog-title">{{'Are you sure you want to replace the existing file?' | rpxTranslate}}</h2>
   </div>
   <div class="dialog-info">
-    <span class="text-info">You are about to delete the original file uploaded. Are you sure you want to proceed?</span>
+    <span class="text-info">{{'You are about to delete the original file uploaded.Are you sure you want to proceed?' | rpxTranslate}}</span>
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/dialogs/document-dialog/document-dialog.component.html:9]: @DavidJayakumar - the space between the full stop and the next sentence should have been preserved

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/base-field/field-read.component.spec.ts [hunk 2]
```diff
@@ -12,13 +12,15 @@ import { CaseFieldService } from '../../../services/case-fields/case-field.servi
...
 import { FormValueService } from '../../../services/form/form-value.service';
+import { MockRpxTranslatePipe } from '../../../test/mock-rpx-translate.pipe';
 import { WizardPage } from '../../case-editor/domain/wizard-page.model';
 import { Wizard } from '../../case-editor/domain/wizard.model';
 import { PageValidationService } from '../../case-editor/services/page-validation.service';
 import { PaletteService } from '../palette.service';
 import { FieldReadComponent } from './field-read.component';
 import { PaletteContext } from './palette-context.enum';
 
+
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/base-field/field-read.component.spec.ts:23]: @DavidJayakumar - what's this extra blank line for?

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.component.spec.ts [hunk 3]
```diff
@@ -4,11 +4,12 @@ import { FormGroup } from '@angular/forms';
 import { MatDialog, MatDialogConfig, MatDialogRef } from '@angular/material/dialog';
 import { By } from '@angular/platform-browser';
 import { MockComponent } from 'ng2-mock-component';
-import { of, Subscription, throwError } from 'rxjs';
+import { Subscription, of, throwError } from 'rxjs';
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.component.spec.ts:7]: @DavidJayakumar - import ordering... are you sure this is correct? Expected to see `{ of, Subscription, throwError }`

File: yarn-audit-known-issues [hunk 4]
```diff
@@ -1,9 +1,14 @@
...
+➤ YN0001: TypeError [ERR_INVALID_URL]: Invalid URL
```
Comment (Daniel-Lam) [inline yarn-audit-known-issues:1]: @DavidJayakumar - this `yarn-audit-known-issues` looks completely wrong

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html [hunk 5]
```diff
@@ -2,23 +2,26 @@
...
+        {{"Contact us" | rpxTranslate}}</a> {{"if you're still having problems." | rpxTranslate}}
+    </p>
   </div>
 </div>
 <!-- Callback error heading and error message to be displayed if there are specific error details -->
 <div *ngIf="error && error.details" class="error-summary" role="group"
      aria-labelledby="edit-case-event_error-summary-heading" tabindex="-1">
   <h2 class="heading-h2 error-summary-heading" id="edit-case-event_error-summary-heading">
-    The callback data failed validation
+    {{"The callback data failed validation" | rpxTranslate}}
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html:19]: @DavidJayakumar - message text should be in single quotes

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-specific-access-request/case-specific-access-request.component.html [hunk 6]
```diff
@@ -1,16 +1,17 @@
 <exui-error-message
   *ngIf="formGroup.invalid && submitted"
   [error]="errorMessage"></exui-error-message>
-<cut-alert type="information"
-  >Authorisation is needed to access this case.<br />This could be because it's outside your jurisdiction, or you may be excluded from the case.
-    If you request access to this case, it will be logged for auditing purposes.</cut-alert>
+<cut-alert type="information">
+  {{'Authorisation is needed to access this case.' | rpxTranslate}}<br />
+  {{"This could be because it's outside your jurisdiction, or you may be excluded from the case.  If you request access to this case, it will be logged for auditing purposes." | rpxTranslate}}
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-specific-access-request/case-specific-access-request.component.html:6]: @DavidJayakumar - text should be in single quotes

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@DavidJayakumar - some changes needed

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-specific-access-request/case-specific-access-request.component.html (hunk 6 above)
Comment (johnbenjamin-hmcts) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-specific-access-request/case-specific-access-request.component.html:6]: @Daniel-Lam the sentence already has a single quote `it's`, so we surrounded it with double quotes.

Comment (DavidJayakumar) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-specific-access-request/case-specific-access-request.component.html:6]: done

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/dialogs/delete-or-cancel-dialog/delete-or-cancel-dialog.component.html [hunk 7]
```diff
@@ -1,16 +1,15 @@
...
     <div class="dialog-header">
       <h2 (click)="cancel()" class="heading-h2 x">X</h2>
     </div>
     <div>
-      <h2 class="heading-h2 dialog-title">Are you sure you want to delete this draft?</h2>
+      <h2 class="heading-h2 dialog-title">{{'Are you sure you want to delete this draft?' | rpxTranslate}}</h2>
     </div>
     <div class="dialog-info">
-      <span class="text-info">You are about to delete this draft. Are you sure you want to proceed?</span>
+      <span class="text-info">{{'You are about to delete this draft.Are you sure you want to proceed?' | rpxTranslate}}</span>
```
[the remark this answers is not in the document]
Comment (DavidJayakumar) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/dialogs/delete-or-cancel-dialog/delete-or-cancel-dialog.component.html:9]: done

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html (hunk 5 above)
Comment (DavidJayakumar) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html:19]: done

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.component.spec.ts (hunk 3 above)
Comment (DavidJayakumar) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.component.spec.ts:7]: It rearragnes atomatically on save. So it seem like correct ordering

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/dialogs/document-dialog/document-dialog.component.html (hunk 1 above)
Comment (DavidJayakumar) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/dialogs/document-dialog/document-dialog.component.html:9]: done

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.component.spec.ts (hunk 3 above)
Comment (Daniel-Lam) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.component.spec.ts:7]: OK... must be a variation between VSCode and IntelliJ, @DavidJayakumar

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/base-field/field-read.component.spec.ts (hunk 2 above)
Comment (DavidJayakumar) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/base-field/field-read.component.spec.ts]: Removed extra line

Comment (DavidJayakumar) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/base-field/field-read.component.spec.ts]: Done

**Review by Daniel-Lam — DISMISSED** [review]

**Review by Daniel-Lam — APPROVED** [review]
@DavidJayakumar - only change since last approval is merge of Release 6.18.2 into this branch; approved

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by Daniel-Lam — APPROVED** [review]
@DavidJayakumar @johnbenjamin-hmcts - approved

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by Mo-Lala84 — APPROVED** [review]
aprpoved

**Review by Daniel-Lam — APPROVED** [review]
@johnbenjamin-hmcts - approved after a quick check with you on the removal of `rpxTranslate`

**Pull request merged**